tests/electron/modules/compliments_spec.js

```diff
@@ -0,0 +1,68 @@
+const helpers = require("../helpers/global-setup");
+
+describe("Compliments module", () => {
+	/**
+	 * move similar tests in function doTest
+	 *
+	 * @param {Array} complimentsArray The array of compliments.
+	 */
+	const doTest = async (complimentsArray) => {
+		await helpers.getElement(".compliments");
+		const elem = await helpers.getElement(".module-content");
+		expect(elem).not.toBe(null);
+		expect(complimentsArray).toContain(await elem.textContent());
+	};
+
+	afterEach(async () => {
+		await helpers.stopApplication();
+	});
+
+	describe("parts of days", () => {
+		it("Morning compliments for that part of day", async () => {
+			await helpers.startApplication("tests/configs/modules/compliments/compliments_parts_day.js", "01 Oct 2022 10:00:00 GMT");
+			await doTest(["Hi", "Good Morning", "Morning test"]);
+		});
+
+		it("Afternoon show Compliments for that part of day", async () => {
+			await helpers.startApplication("tests/configs/modules/compliments/compliments_parts_day.js", "01 Oct 2022 15:00:00 GMT");
+			await doTest(["Hello", "Good Afternoon", "Afternoon test"]);
+		});
+
+		it("Evening show Compliments for that part of day", async () => {
+			await helpers.startApplication("tests/configs/modules/compliments/compliments_parts_day.js", "01 Oct 2022 20:00:00 GMT");
+			await doTest(["Hello There", "Good Evening", "Evening test"]);
+		});
+	});
+
+	describe("Feature anytime in compliments module", () => {
+		describe("Set anytime and empty compliments for morning, evening and afternoon ", () => {
+			it("Show anytime because if configure empty parts of day compliments and set anytime compliments", async () => {
+				await helpers.startApplication("tests/configs/modules/compliments/compliments_anytime.js");
+				await doTest(["Anytime here"]);
+			});
+		});
+
+		describe("Only anytime present in configuration compliments", () => {
+			it("Show anytime compliments", async () => {
+				await helpers.startApplication("tests/configs/modules/compliments/compliments_only_anytime.js");
+				await doTest(["Anytime here"]);
+			});
+		});
+	});
+
+	describe("Feature date in compliments module", () => {
+		describe("Set date and empty compliments for anytime, morning, evening and afternoon", () => {
+			it("Show happy new year compliment on new years day", async () => {
+				await helpers.startApplication("tests/configs/modules/compliments/compliments_date.js", "01 Jan 2022 10:00:00 GMT");
+				await doTest(["Happy new year!"]);
+			});
+		});
+	});
+
+	describe("remoteFile option", () => {
```

Comment:
Do all tests have to be moved to the electron page? Seems unnecessary....

Reply:
thats more a general question if you want split the small test into 2 or having all together.

We have here a range from
- move only tests to `electron` which needs date/time mocking

to
- move all `e2e` tests back to `electron` section

Reply:
From my understanding (and please correct me if I am wrong) it was that electron had only the tests that opened a real electron window, while e2e didnt had to open a electron window. I personally like that distinction and would like to keep the electron tests at a minimum, since running the tests in e2e doesnt disrupt as often and seems faster...

Reply:
Maybe @sdetweil can give an opinion on how he likes the tests sorted :-)

Reply:
I'm asking because I don't want to change this several times, can live with the decision to minimize electron tests.